tests/functional/test_snap_dcgm.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
+from time import sleep
+
```

Comment:
I think would be nice also having simple tests to check the following things:

- nv-hostengine is up and running
- dcgmi command is working (not sure if it will work in a VM, needs a small investigation). Case it doesn't work checking that the --help command works would be enough
- test all the dcgmproftester to see if at least the help command is working

Reply:
yes, aside from dcgmproftester, which fails if you don't have the right driver installed.
```
$ dcgm.dcgmproftester10 
/snap/dcgm/1/usr/bin/dcgmproftester10: error while loading shared libraries: libcuda.so.1: cannot open shared object file: No such file or directory
```

Reply:
Don't know how to test dcgmproftesters, should we install the lib during the test initialization